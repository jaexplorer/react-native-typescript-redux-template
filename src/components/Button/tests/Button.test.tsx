import React from 'react';
import { View, Text } from 'react-native';
import { render } from '@testing-library/react-native';

export const Nothing = () => {
  return (
    <View>
      <Text>Test</Text>
    </View>
  );
};

test('example test', () => {
  const { baseElement } = render(<Nothing />);
  expect(baseElement).toMatchSnapshot();
});
