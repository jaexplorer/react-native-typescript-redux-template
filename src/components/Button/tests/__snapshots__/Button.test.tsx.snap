// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example test 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View>
      <Text>
        Test
      </Text>
    </View>
  </View>
</View>
`;
